compiler/python/byteir/compile.py

```diff
@@ -264,6 +264,11 @@ def _compile_cpu(
     with open(output_file_path, "w") as f:
         f.write(module.operation.get_asm())
 
+register_byteir_compiler_backend(compiler_fn=_compile_cuda, target="cuda", device="cuda")
```

Comment:
Using decorator is more Pythonic.

Reply:
> Using decorator is more Pythonic.

We need to consider that one could register different targets for the same `compiler_fn`, just like `_compile_cuda_with_ait`.

Reply:
I think split `_compile_cuda_with_ait` into two functions `_compile_cuda_with_ait` and `_compile_cuda_with_ait_aggressive` is ok.

Reply:
> I think split `_compile_cuda_with_ait` into two functions `_compile_cuda_with_ait` and `_compile_cuda_with_ait_aggressive` is ok.

That's great.